pkgs/development/python-modules/mwxml/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, jsonschema
+, nose
+, mwcli
+, mwtypes
+}:
+
+buildPythonPackage rec {
+  pname = "mwxml";
+  version = "0.3.3";
+
+  src = fetchPypi {
```

Comment:
why not `fetchFromGitHub`?

Reply:
Because unfortunately there are no releases/tags on the repo.